backend/HoReD.Tests/Controllers/PatientDataControllerTest.cs

```diff
@@ -26,21 +26,55 @@ public void SetUp()
         }
 
         [Test]
-        public void GetDoctors_ValidCountOfRecords()
+        public void GetClosedDiseaseInfo()
```

Comment:
@IvanovVadym the name of the test's method should be understandble for the others
what are we checking here and some more details
in the future, pay more attention to this